workflow-controller/controller-service/src/main/java/io/hyscale/controller/util/ValidatorUtil.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.controller.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.hyscale.commons.constants.ToolConstants;
+import io.hyscale.commons.exception.HyscaleException;
+import io.hyscale.commons.logger.WorkflowLogger;
+import io.hyscale.controller.exception.ControllerErrorCodes;
+
+public class ValidatorUtil {
+
```

Comment:
Why is it a ValidatorUtil ?
When it is trying to handle error messages ?

Reply:
I kept it as a Util thinking later we might need to add more functionality

Reply:
It is not a util...the contract of the class is more to handle error messages